apps/game/src/layer/traffic.rs

```diff
@@ -509,12 +509,14 @@ fn cluster_jams(map: &Map, problems: Vec<(IntersectionID, Time)>) -> Vec<(Polygo
         }
     }
 
+    // TODO This silently hides jams where we can't calculate the convex hull. The caller just
```

Comment:
Here's an annoying consequence of strict checks and fallible conversions

Reply:
Would it be feasible to have a convex_hull method that returns a tesselation? 

It seems like what would be failing is the `abstreet::Polygon <-> geo::Polygon` conversion, right? 

Maybe being able to tesselate from a `geo::Polygon -> abstreet::Tesselation` rather than `geo::Polygon -> abstreet::Polygon -> abstreet::Tesselation` would help mitigate the issue.

related? feature request: https://github.com/georust/geo/issues/568

Reply:
Actually I'm wrong, all 3 need a Polygon (well, without more work to make some operations handle either Polygon or Tessellation). Keeping this for now.

I wrote a `impl From<geo::Polygon> for Tessellation`. I'll go ahead and commit it, even though it's unused right now.